packages/annotations/src/annotations.ts

```diff
@@ -58,11 +56,34 @@ export function getAnnotationServiceUrl(
 /**
  * Query for annotations from ArcGIS Hub.
  * @param requestOptions - request options that may include authentication
- * @returns A Promise that will resolve with features from the annotation service for a Hub enabled ArcGIS Online organization.
+ * @returns A Promise that will resolve with decorated features from the annotation service for a Hub enabled ArcGIS Online organization.
  */
 
 export function searchAnnotations(
   requestOptions: IQueryFeaturesRequestOptions
-): Promise<IQueryFeaturesResponse> {
-  return queryFeatures(requestOptions);
+): Promise<any /* make it a type */> {
+  return queryFeatures(requestOptions).then(response => {
+    const users: string[] = [];
+
+    if (response.features && response.features.length > 0) {
+      response.features.forEach(function(comment) {
```

Comment:
this feels like a good use case for `reduce()` to me, but I'm not going to micromanage your choice of array operations (this time).